store/sqlstore/preference_store.go

```diff
@@ -216,13 +216,16 @@ func (s SqlPreferenceStore) GetAll(userId string) store.StoreChannel {
 	})
 }
 
-func (s SqlPreferenceStore) PermanentDeleteByUser(userId string) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		if _, err := s.GetMaster().Exec(
-			`DELETE FROM Preferences WHERE UserId = :UserId`, map[string]interface{}{"UserId": userId}); err != nil {
-			result.Err = model.NewAppError("SqlPreferenceStore.Delete", "store.sql_preference.permanent_delete_by_user.app_error", nil, err.Error(), http.StatusInternalServerError)
-		}
-	})
+func (s SqlPreferenceStore) PermanentDeleteByUser(userId string) *model.AppError {
+	if _, err := s.GetMaster().Exec(
```

Comment:
Maybe it increases readability if this is written like:
```go
query := `DELETE FROM 
			Preferences 
		WHERE 
			UserId = :UserId`
if _, err := s.GetMaster().Exec(query); err != nil {
```

Reply:
@Wipeout55 Would you please fix this and then we can merge this PR?

Reply:
Done! Sorry for being slow.